src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LocalVariables.cs

```diff
@@ -83,16 +76,6 @@ public override int GetHashCode()
             }
             return Parameter.GetHashCode() ^ Index.GetHashCode();
         }
-
-        public static bool operator ==(LocalDefinition self, LocalDefinition other)
-        {
-            return self.Index == other.Index && self.Parameter == other.Parameter;
-        }
-
-        public static bool operator !=(LocalDefinition self, LocalDefinition other)
-        {
-            return self.Index != other.Index || self.Parameter != other.Parameter;
-        }
```

Comment:
I'd been wary of removing these in a previous cull of dead code, though searching finds a handful of cases where `LocalDefinition?` is compared with literal null, which of course becomes a check on `HasValue` whether these are defined or not. That @ericstj's tool finds it to be dead adds confidence in that.